src/etl/disease_etl.py

```diff
@@ -135,304 +143,315 @@ class DiseaseETL(ETL):
-            [DiseaseETL.execute_annotation_xrefs_template, commit_size, "disease_annotation_xrefs_data_" + \
-             sub_type.get_data_provider() + ".csv"]
+        # This needs to be in this format (query_template, param1, params2) others will be ignored
+        query_template_list = [
+            [self.execute_allele_query_template, commit_size,
+             "disease_allele_data_" + sub_type.get_data_provider() + ".csv"],
+            [self.execute_allele_gene_dej_relationship_query_template, commit_size,
+             "disease_allele_gene_relation_data_" + sub_type.get_data_provider() + ".csv"],
+            [self.execute_gene_query_template, commit_size,
+             "disease_gene_data_" + sub_type.get_data_provider() + ".csv"],
+            [self.execute_agms_query_template, commit_size,
+             "disease_agms_data_" + sub_type.get_data_provider() + ".csv"],
+            [self.execute_pges_gene_query_template, commit_size,
+             "disease_pges_gene_data_" + sub_type.get_data_provider() + ".csv"],
+            [self.execute_pges_allele_query_template, commit_size,
+             "disease_pges_allele_data_" + sub_type.get_data_provider() + ".csv"],
+            [self.execute_pges_agm_query_template, commit_size,
+             "disease_pges_agms_data_" + sub_type.get_data_provider() + ".csv"],
+            [self.execute_withs_query_template, commit_size,
+             "disease_withs_data_" + sub_type.get_data_provider() + ".csv"],
+            [self.execute_ecode_query_template, commit_size,
+             "disease_evidence_code_data_" + sub_type.get_data_provider() + ".csv"],
+            [self.execute_annotation_xrefs_query_template, commit_size,
+             "disease_annotation_xrefs_data_" +  sub_type.get_data_provider() + ".csv"]
         ]
 
         # Obtain the generator
         generators = self.get_generators(data, batch_size, sub_type.get_data_provider())
-        
-        query_and_file_list = self.process_query_params(query_list)
+
+        query_and_file_list = self.process_query_params(query_template_list)
         CSVTransactor.save_file_static(generators, query_and_file_list)
         Neo4jTransactor.execute_query_batch(query_and_file_list)
 
     def get_generators(self, disease_data, batch_size, data_provider):
+        '''Creating generators'''
+
         gene_list_to_yield = []
         allele_list_to_yield = []
         evidence_code_list_to_yield = []
         agm_list_to_yield = []
-        withs =[]
+        withs = []
         pge_list_to_yield = []
         xrefs = []
         counter = 0
-        dateProduced = disease_data['metaData']['dateProduced']
+        date_produced = disease_data['metaData']['dateProduced']
 
-        dataProviders = []
+        data_providers = []
 
-        dataProviderObject = disease_data['metaData']['dataProvider']
+        data_provider_object = disease_data['metaData']['dataProvider']
 
-        dataProviderCrossRef = dataProviderObject.get('crossReference')
```

Comment:
Early on, we agreed to use camel case in the python ETL scripts, not because we were trying to go against python conventions, but rather because the downstream components of our software stack prefer them.  ie: in neo, we have an agreed up convention of camel casing attributes and node names.  Changing everything to patch the python convention of underscores makes sense from a code POV, but from a full project POV it makes the translation between variable conventions in neo vs the code up to the developer - and importantly, the IDE doesn't catch camel case  vs. underscore issues in the cypher queries because they are just long strings in python.  (ie: if I write a cypher query that looks for 'row.primaryId' my ID won't catch the error (ie: should be row.primary_id).  This makes me  think that this is a more error prone convention.  ie: in this file all of the python variables are underscored now, whereas the maps used to feed into the CSV generator (and eventually neo) are all still camel case.  Further, the attribute setting in the cypher queries are still camel case (as they should be).  I don't want to debate camel case versus underscore casing in depth, because I don't actually feel that strongly about it - but, I did want to point out that we did have a standard for this already, and it was there for a reason (and it keeps the variable names consistent).

Reply:
Hi @sierra-moxon I have had many similar thoughts. I had not remembered the official decision we came up with regards to camelcase vs underscore. I agree with there are limitations with IDEs finding issues with variables in the Cypher queries. It is definitely awkward how there are two styles that are very commonly used in many languages and databases.